Kudu.Contracts/Diagnostics/ProcessModuleInfo.cs

```diff
@@ -1,49 +1,49 @@
 ﻿using System;
 using System.Diagnostics.CodeAnalysis;
 using Kudu.Contracts.Infrastructure;
-using Newtonsoft.Json;
```

Comment:
I like the idea of not using Newtonsoft.Json but please make sure we run KuduFunctionals to make sure we don't regress any functionality

Reply:
Functional Tests on the main branch without my changes seem to be failing for me. Coordinating with suwat to see if I can find a way to run these.